### PR TITLE
Fix a few incorrect strings

### PR DESCRIPTION
**Pull request by nn9dev**
I know... another language pr...
This corrects a few log things and GUI error warning screens. Just makes things a tad cleaner.

### CODE REVIEW

File: vita3k/gui/src/archive_install_dialog.cpp [hunk 1]
```diff
@@ -183,15 +183,15 @@ void draw_archive_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
                     });
                 };
                 ImGui::Spacing();
-                ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%lu archive(s) found with compatible contents.", contents_archives.size());
+                ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%lu archive(s) found with compatible content.", contents_archives.size());
```
Comment (Zangetsu38) [inline vita3k/gui/src/archive_install_dialog.cpp:186]: revert, is full normal, archive should have few contents

File: vita3k/gui/src/archive_install_dialog.cpp [hunk 2]
```diff
@@ -183,15 +183,15 @@ void draw_archive_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
...
                 ImGui::Spacing();
                 ImGui::Separator();
                 for (const auto &archive : contents_archives) {
                     ImGui::TextWrapped("%s", string_utils::wide_to_utf(archive.first.filename().wstring()).c_str());
                     ImGui::Spacing();
                     const auto count_contents_successed = count_content_state(archive.first, true);
                     if (count_contents_successed) {
-                        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%ld contents of this archive have successed installed:", count_contents_successed);
+                        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%ld items from this archive have been successfully installed:", count_contents_successed);
```
Comment (Zangetsu38) [inline vita3k/gui/src/archive_install_dialog.cpp:194]: keep contents, is not items, is contents

File: vita3k/gui/src/archive_install_dialog.cpp [hunk 3]
```diff
@@ -203,11 +203,11 @@ void draw_archive_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
                     const auto count_contents_failed = count_content_state(archive.first, false);
                     if (count_contents_failed) {
                         ImGui::Spacing();
-                        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%ld content have failed installed:", count_contents_failed);
+                        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%ld items have failed to install:", count_contents_failed);
```
Comment (Zangetsu38) [inline vita3k/gui/src/archive_install_dialog.cpp:206]: same

File: vita3k/gui/src/archive_install_dialog.cpp [hunk 4]
```diff
@@ -217,7 +217,7 @@ void draw_archive_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
             }
             if (!invalid_archives.empty()) {
                 ImGui::Spacing();
-                ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%lu archive(s) did no found compatible contents:", invalid_archives.size());
+                ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%lu archive(s) found with compatible content:", invalid_archives.size());
```
Comment (Zangetsu38) [inline vita3k/gui/src/archive_install_dialog.cpp:220]: leep contents

File: vita3k/gui/src/content_manager.cpp [hunk 5]
```diff
@@ -401,9 +401,9 @@ void draw_content_manager(GuiState &gui, EmuEnvState &emuenv) {
             title = "Applications";
             if (gui.app_selector.user_apps.empty()) {
                 ImGui::SetWindowFontScale(1.2f);
-                const auto calc_text = ImGui::CalcTextSize("There are no content items.");
+                const auto calc_text = ImGui::CalcTextSize("There are no items.");
                 ImGui::SetCursorPos(ImVec2((SIZE_LIST.x / 2.f) - (calc_text.x / 2.f), (SIZE_LIST.y / 2.f) - (calc_text.y / 2.f)));
-                ImGui::TextColored(GUI_COLOR_TEXT, "There are no content items.");
+                ImGui::TextColored(GUI_COLOR_TEXT, "There are no items.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/content_manager.cpp:406]: revert this file, other pr fix this 👌

**Comment by Zangetsu38** [comment]
good works :)

**Comment by nn9dev** [comment]
In English there is a slight difference between content and contents. With "content" it can be an ever-changing value, but in a defined space. "Contents" is a countable and known list. 

From [oneminuteenglish.org](https://oneminuteenglish.org/en/content-or-contents/):
> “Content” and “contents,” defined as “an amount contained in a thing,” don’t work like most singular and plural nouns. “Content” [kŏn′tĕnt′] is singular, but it is a singular uncountable noun. “Contents” is plural, but it is known as a plural countable noun.

> It is proper to say “the book’s content” and not “the book’s contents.” The amount of whatever contained in the book is uncountable and thereby singular. Likewise, there is “the water content of our oceans” or “the content of water in our oceans.”

> When you break down the content of a book into countable sections, you will have a “Table of Contents.” When it comes to a box, it is proper to say, “the contents of the box.” The contents are countable or measurable, in weight, a number of items, or volume, thereby plural.

For this reason I have changed the majority of the uses of "contents" to "content", and I believe they should stay. I invite you to ask other native English speakers about this change. As for the ones about "items", I can change them to content(s) as proper English permits.

**Comment by nn9dev** [comment]
I have reviewed the suggestions and asked a few of my peers as to whether "items" or "content(s)" is proper, and they said "items" was the better term to use in this scenario. "Items" refer to individual things, as opposed to "content" or "contents" which imply they are part of a larger whole.

**Comment by EXtremeExploit** [comment]
what about packages?, i agree that contents its not the best wording, items sounds better but its kinda meh when it refers to themes or patches. just an opinion tho

**Comment by Zangetsu38** [comment]
> what about packages?, i agree that contents its not the best wording, items sounds better but its kinda meh when its not referring to themes or patches. just an opinion tho

Package for me is for pkg only ^^ i have used content because content manager 😁

**Comment by Zangetsu38** [comment]
need rebase and fix conflict @nn9dev

**Comment by nn9dev** [comment]
@Zangetsu38 rebased and fixed conflict

**Comment by Zangetsu38** [comment]
ouch, you need learn how rebase, here is totaly failed lol

**Comment by nn9dev** [comment]
should i just redo the pr? I feel like i've made things a lot more complicated 😭

**Comment by Zangetsu38** [comment]
redo ? no juste learn rebase that all, you need update szubmodule before push

**Comment by nn9dev** [comment]
update: git is not my friend tonight... i'm just gonna redo it...

**Comment by Zangetsu38** [comment]
no, do''t create few pr for same things, LEARN git, is very no hard

**Pull request closed without merging**